src/backend/base/langflow/main.py

```diff
@@ -127,6 +128,9 @@ async def lifespan(_app: FastAPI):
             await create_or_update_starter_projects(all_types_dict)
             telemetry_service.start()
             await load_flows_from_directory()
+            queue_service = get_queue_service()
+            if not queue_service._cleanup_task:  # Start if not already started
+                _tasks.append(asyncio.create_task(queue_service.start()))
```

Comment:
```suggestion
                task = asyncio.create_task(queue_service.start())
                _tasks.append(task)
                task.add_done_callback(_tasks.discard)
```
Remaining problem with this is that the task is never awaited...

Reply:
Maybe the task could be referenced locally and awaited in the `finally` block ?

Reply:
I did it like this because it will eventually run but it doesn't have to be in this exact moment. Does that make sense?